src/pages/Download.js

```diff
@@ -33,299 +33,397 @@ const Download = () => {
-                      <li>RAM: minimum 2GB</li>
-                      <li>Display: 1280×768</li>
-                      <li>GPU: minimum 1GB VRAM; OpenGL 3.3+</li>
-                      <li>Hard Drive: minimum 500MB free space</li>
-                    </BulletListInside>
-                  </div>
-                </TabSection>
-                <br />
-                <TabSection>
-                  <div>
-                    <SectionTitle>Other links</SectionTitle>
-                    <BulletListInside>
-                      <li>
-                        Alternative download link&nbsp;
-                        <a
-                          href='https://mega.nz/folder/RsFSSZjA#rTNA_4nBbLp0BSTX9-DqSA'
-                          target={'_blank'}
-                          rel='noreferrer'>
-                          (MEGA)
-                        </a>
-                      </li>
-                      <li>
-                        Download old UPBGE versions from&nbsp;
-                        <a
-                          href='https://github.com/UPBGE/upbge/releases'
-                          target={'_blank'}
-                          rel='noreferrer'>
-                          here
-                        </a>&nbsp;
-                        or&nbsp;
-                        <a
-                          href='https://mega.nz/folder/k9MW1KiZ#UOKzjh3IQ0GEgjQ6GUc7ug'
-                          target={'_blank'}
-                          rel='noreferrer'>
-                          here
-                        </a>
-                      </li>
-                    </BulletListInside>
-                  </div>
-                </TabSection>
+                <Tabs forceRenderTabPanel>
+                  <TabSection>
+                    <TabList>
+                      <Tab>Windows</Tab>
+                      <Tab>Linux</Tab>
+                      <Tab>macOS</Tab>
+                      <Tab>Experimental</Tab>
+                    </TabList>
+                    <TabPanel>
+                      <TabSection>
+                        <div>
+                          <SectionTitle>Windows 8.1, 10, & 11</SectionTitle>
+                          <Button
+                            href={'https://github.com/UPBGE/upbge/releases/download/v0.30/UPBGE-0.30-windows-x86_64.7z'}
+                            target={'_blank'}>Download (64-bit)
+                          </Button>
+                        </div>
+                        <div>
+                        <SectionTitle>Windows 7</SectionTitle>
+                        <Button
+                          href={'https://github.com/UPBGE/upbge/releases/download/v0.30/UPBGE-0.30-windows7-x86_64.7z'}
+                          target={'_blank'}>Download (64-bit)
+                        </Button>
+                        </div>
+                        <div>
+                          <SectionTitle>Requirements</SectionTitle>
```

Comment:
before to each SectionTitle we can add an horizontal separation line (white). What do you think?